src/gui_common/KeyPrompt.tscn

```diff
@@ -2,16 +2,11 @@
 
 [ext_resource path="res://src/gui_common/KeyPrompt.cs" type="Script" id=1]
 
-[node name="KeyPrompt" type="Control"]
-anchor_right = 1.0
-anchor_bottom = 1.0
+[node name="KeyPrompt" type="TextureRect"]
+margin_right = 40.0
+margin_bottom = 40.0
+expand = true
 script = ExtResource( 1 )
 __meta__ = {
 "_edit_use_anchors_": false
 }
-
-[node name="Icon" type="TextureRect" parent="."]
-margin_right = 40.0
-margin_bottom = 40.0
-expand = true
-stretch_mode = 6
```

Comment:
These changes affect how the tutorial looks in a way that I don't like. The key prompts in the tutorial should be bigger.
![2021-07-14_17 18 39 0655](https://user-images.githubusercontent.com/3796411/125637938-3b8eb53b-7752-4811-83fa-683465608808.png)